src/app/work-item/work-item-list/work-item-list-entry/work-item-list-entry.component.scss

```diff
@@ -5,6 +5,10 @@
   cursor: pointer;
   font-size: rem(12);
   align-items: center;
+  .user-avatar{
```

Comment:
Added height: 100%;
Did not add overflow or scroll as the parent div has fixed width and height.